contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -307,41 +307,56 @@ <h4 class="talent_name"></h4>
     </div>
 
     <script>
+        function start(){
+            console.log("receiving data of capability...")
+            ajax('GET', get_cap_url, null,
+                    function (res) {
+                        var items = JSON.parse(res);
+                        console.log("we got a list of capabilities from api: ")
+                        console.log(items)
+                        all_capability = items
+                        append_capability(all_capability);
+                        filterSelection('all');
+                    },
+                    function () {
+                        console.log("error occured");
+                    });
+            console.log("receiving data of talent...")
+            ajax('GET', get_talent_url, null,
+                    function (res) {
+                        var items = JSON.parse(res);
+                        console.log("we got a list of talents from api: ")
+                        console.log(items)
+                        all_talents = items
+                        append_talent(all_talents);
+                        filterSelection('all');
+                    },
+                    function () {
+                        console.log("error occured");
+            });
+            console.log("appending cards... ")
+            window.onload = append_capability(all_capability);
+            window.onload = append_talent(all_talents);
+        }
+
         var all_capability = [];
         var all_talents = []
         var get_talent_url = "http://"  + "localhost:5000/contributions/talents";
         var get_cap_url = "http://" +  "localhost:5000/contributions/capabilities";
 
-        ajax('GET', get_cap_url, null,
-                function (res) {
-                    var items = JSON.parse(res);
-                    console.log("we got some results: ")
-                    console.log(items)
-                    all_capability = items
-                    append_capability(all_capability);
-                },
-                function () {
-                    console.log("error occured");
-                });
-
-       ajax('GET', get_talent_url, null,
-                function (res) {
-                    var items = JSON.parse(res);
-                    console.log("we got some results: ")
-                    console.log(items)
-                    all_talents = items
-                    append_talent(all_talents);
-                },
-                function () {
-                    console.log("error occured");
-        });
 
-       filterSelection("all");
+        console.log("show all ")
```

Comment:
Could we remove the console message and the commented out code?

Reply:
Hi @wenjzhu, It looks like my earlier comment may not have been clear. I was suggesting deleting unwanted console log messages and as well as to delete commented-out code. Thanks.

Reply:
Sorry I realized I misread your comments before...I will work on cleaning commented code as well as console.